pycaret/tests/time_series_test_utils.py

```diff
@@ -107,6 +107,7 @@ def _return_model_names():
         "X_train": pd.DataFrame(get_data("airline")),
         "enforce_pi": False,
         "seasonal_period": 2,
+        "sp_to_use": 2,
```

Comment:
And same here, by definition, `sp_to_use` will at least be 1, so I add it in this fixture.

Reply:
Did you mean `sp_to_use` will be 2 and not 1?

Reply:
Yes, sorry, what I wanted to say really is that `sp_to_use` depends on whether the seasonality check for `seasonal_period=2` is positive (in which case `sp_to_use=2`) or negative (in which case `sp_to_use=1`).

In this case, we know that such check is positive and then `sp_to_use=2`.

Reply:
ok, thanks!